### PR TITLE
Adds new phobias and tweaks others - syndicate, eye phobia

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Makes a syndicate phobia for all those that fear the syndicats
Makes other phobia's more robust at hecking people over
Makes a new eye phobia!
:cl: optional name here
add: syndicate phobia
tweak: other phobia's
/:cl:

[why]
Lots of phobia's are missing new tech or flat out free non-affecting phobia's well other make the game into a paranoia survival space sim like we say it is on the tin

### CODE REVIEW

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -158,25 +160,29 @@ SUBSYSTEM_DEF(traumas)
 						
 					"cats" = typecacheof(list(/obj/item/organ/ears/cat, /obj/item/organ/tail/cat, /obj/item/laser_pointer, /obj/item/toy/cattoy, /obj/item/clothing/head/kitty,
 						/obj/item/clothing/head/collectable/kitty, /obj/item/melee/chainofcommand/tailwhip/kitty, /obj/item/stack/sheet/animalhide/cat))
+						
+					"syndicate"   = typecacheof(list(/obj/item/stack/tile/mineral/plastitanium, /obj/item/aiModule/, /obj/machinery/recharge_station,
+						/obj/item/aicard, /obj/item/deactivated_swarmer, /obj/effect/mob_spawn/swarmer)),
```
Comment (Useroth) [inline code/controllers/subsystem/traumas.dm]: Not entirely sure about these.
`/obj/item/aicard, /obj/item/aiModule/, /obj/machinery/recharge_station`

Comment (Trilbyspaceclone) [reply code/controllers/subsystem/traumas.dm]: place holders

File: strings/phobia.json [hunk 2]
```diff
@@ -283,6 +296,25 @@
...
+		"ops",
+		"traitor",
+		"second shooter",
+		"revolution",
+		"fuck nt",
+		"rigged from the start",
+		"det to pay",
+		"brainwashing",
+		"enemy corporation",
+		"angent"
```
Comment (deathride58) [inline strings/phobia.json]:
```suggestion
		"agent"
```

File: strings/phobia.json [hunk 3]
```diff
@@ -283,6 +296,25 @@
...
+		"nuke ops",
+		"syndicats",
+		"nukies",
+		"ops",
+		"traitor",
+		"second shooter",
+		"revolution",
+		"fuck nt",
+		"rigged from the start",
+		"det to pay",
```
Comment (deathride58) [inline strings/phobia.json]:
```suggestion
		"debt to pay",
```

File: strings/phobia.json [hunk 4]
```diff
@@ -283,6 +296,25 @@
 		"slip"
 	],
 	
+	"syndicate": [
+		"syndicate",
+		"changling",
```
Comment (deathride58) [inline strings/phobia.json]:
```suggestion
		"changeling",
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Review by deathride58 — APPROVED** [review]

**Pull request merged**